### ISSUE TITLE
From what I can see - there is no strong typing on Veggie pizza...

### ISSUE THREAD

**Issue by computamike** [MachOneSoftware/PizzaBuddy#15]
The pre-made pizza (beef lovers pizza for example) are collections of strings as their toppings, whereas there nothing preventing a person from putting anchovies onto a veggie pizza, or as part of the 'recipe' 

so - for instance 

{ "Meat Lovers",        new[] { "pepperoni", "beef", "sausage", "ham", "Soilent Green"} },

is a technically correct, because a meat pizza is a `Lazy<Dictionary<string, string[]>` whereas it should probably be : 

`Lazy<Dictionary<string, _meats[]>` 
(not sure of syntax here)
 
Perhaps these should be refactored, so that a pizza has a list of toppings of a type, preventing a shop from offering a veggie pizza with accidental meat.  If should also mean that from a type safety point of view that it wouldn't be possible to add an ingredient to a pizza without it existing already in the Meat / veggies type.

**Comment by javajudt** [comment]
Good point. Part of why I implemented it this way is because the response that Alexa speaks comes directly from the array of strings. It was simple to implement it this way because I could just do a string.Join() and inject directly into the OutputSpeech. But your idea leads to more robust code. I think it's similar to #30 that I just opened.
